docs/repositories-configure/adjusting-quality-gates.md

```diff
@@ -1,6 +1,11 @@
 # Adjusting quality gates
 
-Adjust the **quality gates** to configure when Codacy reports your pull requests and commits as not up to standards.
+The **quality gates** of your repository configure when Codacy reports your pull requests and commits as not up to standards.
+
+When you add your repository to Codacy, it automatically follows the [default gate policy for your organization](../organizations/using-gate-policies.md). If you want to set different quality gates for the repository, [create a new organization gate policy](../organizations/using-gate-policies.md#creating) to apply to the repository.
+
+!!! important
```

Comment:
Could this be a `note` or even `tip` admonition instead? Like this, I feel like it could give the impression that users can potentially hit some issues (like bugs) if they use the repository-level settings. :thinking:

Reply:
Creating a new gate policy to have different quality gates is the recommended behavior. If the user sets different repository-level settings, the product shows the following warning: 

![image](https://github.com/codacy/docs/assets/26439949/e7ae6908-f50f-4598-a856-ef77af83093c)

Thus, I wouldn't include this expected behavior in an admonition. WDYT? 🤔

Reply:
Makes more sense now! 😄 
I changed it into a note.
Also, I changed the next one into a tip, as it seems more appropriate and avoids having two **notes** so close. Let me know if you agree.

Reply:
Nice tweak! :raised_hands: